tests/test_derivatives.py

```diff
@@ -261,6 +262,56 @@ def test_fd_space(self, derivative, space_order):
                     Dpolyvalues[space_border:-space_border])
         assert np.isclose(np.mean(error), 0., atol=1e-3)
 
+    @pytest.mark.parametrize('staggered', [(True, True), (False, False),
+                                           (True, False), (False, True)])
+    @pytest.mark.parametrize('space_order', [2, 4, 6, 8, 10, 12, 14, 16, 18, 20])
+    def test_fd_space_45(self, staggered, space_order):
```

Comment:
We should test this in 3D as well to make sure everything is implemented as intended